src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -32288,6 +32288,20 @@ unit:NUM-PER-MilliGM
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Number per milligram"@en .
 
+unit:NUM-PER-MilliL
+  a qudt:Unit ;
+  dcterms:description "Count of an entity or phenomenon occurrence in one thousandth of the SI unit of volume (litre)."@en ;
+  qudt:conversionMultiplier 0.000001 ;
```

Comment:
This should be 1000000.0

Reply:
oh, yes.

Reply:
The shacl check should have caught that

Reply:
Agreed. I'll add a shape.

Reply:
fixed

Reply:
sorry since its 1/0.000001, I have some more to do so I get there with the logic that you used.

Reply:
I just tell myself "If it is 1 count per MilliL, then how many counts per cubic metre? That's the conversion multiplier.